greeter/README.md

```diff
@@ -681,20 +690,19 @@ greeter-extension Sweeping cache... +10s
 Try endpoint GET /greet/{name} again from explorer, see its result
 Go back to the console window, after 10s, you will see the cached result gets swept
 
-```s
+```
 Cache for en:/greet/loopback is swept.
 ```
 
-Summary
 The interceptor, service and observer could be contributed by from different modules, or packaged together as a component, all depends on your choice. You just bind them to the app without modifying the business logic of your endpoint.
 
-Back to the concepts, now audience should have a more through understanding of how to "provide aspect-oriented logic around method invocations" and how to "allow artifacts to participate in the life cycles".
-
 ## Conclusion
 
 Congratulations! You have completed the tutorial.
 
-You have successfully built a LoopBack application using an existing extension `greeter-extension`, and then you have added functionality on top of the existing extension, i.e. add a French greeter. Lastly, you have enabled caching as part of the application lifecycle using a lifecycle observer.
+First, you built a LoopBack application using a component GreetingComponent located in a different module named greeter-extension. Then you extended the functionality to the GreetingComponent by adding FrenchGreeter as an extension. Then you created a caching service that is started and stopped by a lifecycle observer. And lastly, you retrieved/stored request responses from/in the cache by using an interceptor.
+
+Now you should have a more through understanding of how to decouple your app and design it as microservice with LoopBack4!
```

Comment:
> a more through understanding 

a more `thorough` understanding

Reply:
Feel free to just edit it if its just repo/rewording so that we don't have to go back and forth :)